src/redux/store.js

```diff
@@ -1,9 +1,12 @@
 import { configureStore } from '@reduxjs/toolkit';
 import MoviesSlice from './MoviesSlice';
+// eslint-disable-next-line import/extensions
+import userReducer from './UsersSlice';
```

Comment:
Hello @SabaAhmad404  and @Lucash2022 
Kindly ensure the `userReducer` is imported from the correct file path in order for the app to work as intended

Reply:
Hello @torobucci,

Thank you so much for taking the time to review our code and providing valuable feedback. Your keen eye helped us identify the issue with the userReducer import, and we truly appreciate your input.

We have carefully looked into the problem you pointed out and have successfully resolved it by renaming the file from Userslice.js to UsersSlice.js. This change has been committed, and you can find the details in this commit: [Commit Link](https://github.com/Lucash2022/BookFlix-Frontend/pull/16/commits/146f4890ad5a091fc606b451f94e821cf605f2d5).

Thanks to your review, the app is now working as intended, and the userReducer is imported correctly.

Once again, we are grateful for your help in improving our project. Your feedback has been invaluable to us. If you have any more suggestions or feedback in the future, please don't hesitate to share it with us. We look forward to working together to enhance our codebase.

Best regards,
@Lucash2022 and @SabaAhmad404